src/pages/locationsettings/index.tsx

```diff
@@ -0,0 +1,251 @@
+import React, { useState, useEffect, useRef, ChangeEvent } from "react";
+import Layout from "~/components/Layout";
+import { ToastContainer, toast } from "react-toastify";
+
+const index = () => {
+  interface IStadt {
+    name: string;
+    population: number;
+  }
+
+  const cities = [
```

Comment:
Assign the interface to the object
```suggestion
  const cities: ICities = [
```